Generator/ConfigurationGenerator.php

```diff
@@ -26,7 +26,9 @@ public function generateConfiguration(array $objectDefinitions)
 
         foreach ($this->transformers as $transformer) {
             $inputData = count($transformedData) ? $transformedData: $objectDefinitions;
-
+            if (empty($inputData)) {
```

Comment:
Would be more logical to check length of `$objectDefinitions` outside the loop. Only the first passed array matters. :)